flow360/component/simulation/unit_system.py

```diff
@@ -35,6 +35,9 @@
 )
 u.dimensions.inverse_area = 1 / u.dimensions.area
 u.dimensions.inverse_length = 1 / u.dimensions.length
+u.dimensions.mass_flux = u.dimensions.mass / u.dimensions.time
```

Comment:
We shouldn't call this flux, flux is usually per area. mass_flow_rate?

Reply:
Done